### DISCUSSION TITLE
トラックボールを支持しているベアリングが取れてしまった

### DISCUSSION THREAD

**Discussion by CheapHead** [Yowkees/keyball#602] [Q&A]
いつもサポートありがとうございます。
トラックボールがぐらつく感じと引っかかる感じがして、ケースを確認したところ向かって右側、下側のベアリングボールがなくなっているように思いました。
こちらは交換パーツなどがあるでしょうか。
トラックボールケース全体の交換とかできたりしますでしょうか。
お忙しい中すみませんが回答くださいますようお願いします。
![image](https://github.com/Yowkees/keyball/assets/34470444/9bef22e4-a8f2-410c-91b8-eb2df3337bb0)
![image](https://github.com/Yowkees/keyball/assets/34470444/e6e86d4b-6ec5-4fe5-8af8-56dfe41b0f02)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

トラックボールケースに取り付けているセラミックボールについて、白銀ラボのウェブショップから購入者様へ向けてセラミックボール、トラックボールケースの販売を行っております。
どちらの販売もできますので、お問い合わせフォームからご連絡いただければと思います。

**Reply by CheapHead** [reply]
回答ありがとうございます。
教えていただいた手順で無事パーツ購入できました。
とても助かりました。
